velox/dwio/parquet/writer/arrow/Properties.cpp

```diff
@@ -23,9 +23,10 @@
 
 #include "arrow/io/buffered.h"
 #include "arrow/io/memory.h"
-#include "arrow/util/logging.h"
 #include "arrow/util/thread_pool.h"
 
+#include "velox/common/base/Exceptions.h"
```

Comment:
nit: Is this header used?

Reply:
This and other `#include` are needless. Sorry. I missed them.